docs/cost/index.md

```diff
@@ -89,12 +80,112 @@ The following mapping tables describe:
 ```eval_rst
 .. csv-table-no-translate::
    :header-rows: 1
-   :widths: 20 20 30 30
+   :widths: 20 20 20 30
    :file: ../../build/current_lang/process-level-procurement.csv
 ```
 
 #### Implementation
 
+```eval_rst
+.. csv-table-no-translate::
+   :header-rows: 1
+   :widths: 20 20 20 30
+   :file: ../../build/current_lang/process-level-implementation.csv
+```
+
+## OCDS to OC4IDS Mapping
+
+### Guidance
+
+#### Command-line tool and reference implementation
+
+OCDS Kit's [convert-to-oc4ids](https://ocdskit.readthedocs.io/en/latest/cli/ocds.html#convert-to-oc4ids) command  is a command-line tool and reference implementation for converting OCDS data to OC4IDS format.
+
+`convert-to-oc4ids` covers most mappings in the following categories:
+
+* project-level identification
+* project-level preparation
+* process-level procurement
+
+However, `convert-to-oc4ids` does not cover all mappings, nor does it perform currency conversions. Mappings which `convert-to-oc4ids` does not cover are shown in *italics*.
+
+#### Mapping codelists
+
+Mappings that depend the specific classifications or codelists used in the OCDS data are not documented in detail, as they may differ by publisher. For example, mapping to the OC4IDS projectSector codelist.
```

Comment:
There might be a word missing here ("depend the")

Reply:
Fixed in https://github.com/open-contracting/infrastructure/pull/294/commits/cbe93f3c8ecaf082d9dfdf75eba16dee150e714b